src/Controls/src/Core/Registrar.cs

```diff
@@ -412,6 +414,13 @@ public static void RegisterAll(Type[] attrTypes, InitializationFlags flags)
 				Profile.FrameEnd(frameName);
 			}
 
+			if (FontRegistrar is FontRegistrar fontRegistrar)
+			{
+				var type = Registered.GetHandlerType(typeof(EmbeddedFont));
+				if (type != null)
+					fontRegistrar.SetFontLoader((IEmbeddedFontLoader)Activator.CreateInstance(type));
+			}
```

Comment:
This is interesting. Once I have registered all the fonts and all the "handlers", I then need to let the registrar know what the handler is to use when installing the fonts.